pkg/cmd/registry/artifact/.notes/References.md

```diff
@@ -0,0 +1,63 @@
+# Artifact References
+
+## Usage Example:
+
+First artifact:
+
+```
+echo '{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://example.com/schema1.json",
+    "type": "object",
+    "$defs": {
+        "name": {
+            "type": "string"
+        }
+    }
+}' > artifact1.json
+```
+
+Second artifact:
+
+```
+echo '{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://example.com/schema2.json",
+    "type": "object",
+    "properties": {
+        "external": {
+            "$ref": "schema1.json#/$defs/name"
+        }
+    },
+    "$defs": {
+        "personal": {
+            "email": {
+                "type": "string",
+                "format": "email"
+            },
+            "birthday": {
+                "type": "string",
+                "format": "date"
+            }
+        }
+    }
+}' > artifact2.json
+```
+
+Create first artifact:
+
+```
+cat artifact1.json | ./rhoas service-registry artifact create
+```
+
+Create second artifact:
+
+```
+cat artifact2.json | ./rhoas service-registry artifact create -r 'https://example.com/schema1.json#/$defs/name=:0430a24f-ae35-485b-ada8-61376f1286d7:'
```

Comment:
When someone will reproduce this scenario, they will encounter an error, because their artifacts will have different artifact-id. I suggest adding information about that in the example.

+ I would personally love to see the outputs in the example since it would make the example much more clear for me.